### ISSUE TITLE
UIApplicationStateBackground : didDiscoverBeacons not called 

### ISSUE THREAD

**Issue by Alkashi** [smartbeacon/smartbeacon-base-ios-sdk#1]
Hi,

1/ I launch my app
2/ I press the home button and lock the phone
3/ didDiscoverBeacons is never called and I can't see any local notification :/

When app is in active state, everything work fine and I can see all my beacons stations.

**Comment by smartbeacon** [comment]
Hello,
Do you have enable "location updates" and "uses BLE accessories" background modes in "capabilities" tab of your xcode project?

**Comment by Alkashi** [comment]
Yes. Ever done. Note solved the issue. When the device is locked I receive no notification.
However, I receive the notifications regarding fired on the didEnterRegion and didExitRegion methods.

The issue concerned only the didDiscoverBeacons method.

**Comment by Alkashi** [comment]
Hello,

Any news about that ?

Thx.
